### PR TITLE
Updates to large_image work

### PR DESCRIPTION
**Pull request by banesullivan**
Continuation of #276 

See https://github.com/ResonantGeoData/ResonantGeoData/pull/276#issuecomment-789151006 for info

### CODE REVIEW

File: rgd/geodata/templates/geodata/rastermetaentry_detail.html [hunk 1]
```diff
@@ -55,54 +55,24 @@
...
 
     function updateBandThumbnail () {
       selector = document.getElementById('band-selector')
       var image_id = Number(selector.value);
-      let image = null;
+      console.log(image_id)
       if (image_id >= 0) {
-        var src = extents['thumbnails'][image_id];
-        image = createImage(src);
+        tiles.url(`http://localhost:8000/api/geodata/imagery/image_entry/${image_id}/tiles/{z}/{x}/{y}.jpg`)
```
Comment (banesullivan) [inline rgd/geodata/templates/geodata/rastermetaentry_detail.html:67]: We need to make sure we are not hardcoding the host (e.g. `http://localhost:8000/`) but pulling it from the context. How should I go about this?

Comment (banesullivan) [reply rgd/geodata/templates/geodata/rastermetaentry_detail.html:67]:
```js
var host = window.location.protocol + "//" + window.location.host;
```

File: rgd/geodata/urls.py [hunk 2]
```diff
@@ -110,8 +110,13 @@
         name='tilemetadata',
     ),
     path(
-        'api/geodata/imagery/image_entry/<int:pk>/tiles/<int:z>/<int:x>/<int:y>.jpg',
+        'api/geodata/imagery/image_entry/<int:pk>/tiles/<int:z>/<int:x>/<int:y>.png',
```
Comment (mcovalt) [inline rgd/geodata/urls.py:113]: We're pulling in a JPEG every time, though, right?

Comment (banesullivan) [reply rgd/geodata/urls.py:113]: I changed the encoding to PNG in https://github.com/ResonantGeoData/ResonantGeoData/pull/284/commits/981a7d124412d3a4e30fa04771ae31fa8d8a2c2d with:

```py
tile_source = GDALFileTileSource(file_path, projection=projection, encoding='PNG')
```

File: rgd/geodata/tests/test_tiles.py [hunk 3]
```diff
@@ -18,14 +18,21 @@ def image_entry():
 def test_metadata(api_client, image_entry):
     response = api_client.get(f'/api/geodata/imagery/image_entry/{image_entry.pk}/tiles')
     metadata = response.data
-    assert metadata['levels'] == 2
+    assert metadata['levels'] == 15
```
Comment (banesullivan) [inline rgd/geodata/tests/test_tiles.py:21]: @mcovalt, when I changed the format to PNG this changed to 15. Does that seem okay?

**Comment by banesullivan** [comment]
I'm going to merge this into #276 to put all of this in one place

**Pull request merged**